project.clj

```diff
@@ -7,8 +7,8 @@
 
   :source-paths ["src/clj" "src/cljs"]
 
-  :dependencies [[org.clojure/clojure "1.6.0" :scope "provided"]
-                 [org.clojure/clojurescript "0.0-2371" :scope "provided"]
+  :dependencies [[org.clojure/clojure "1.8.0" :scope "provided"]
```

Comment:
I guess, I could revert the clojure bump.

About cljs: Google's XhrIo lib got support for these events only [in August 2015](https://github.com/google/closure-library/commit/42162f3d5ccbb2f59e3c8a9780ec2924e919dd91), so I assume it's in the v20150901 release (there is no changelog entry). Clojurescript bumped the version in [October 2015](https://github.com/clojure/clojurescript/commit/d2fbb1364bb60aa32249f1b003a4899284c7ea4c), which was included in the 1.7.170 release.

I'm not that experienced with clojurescript, so I trust you which version I should require here. I'm not even sure, whether this has effects on the users of the lib. :wink: